doc/quickstart.rst

```diff
@@ -18,10 +18,249 @@ examples below:
 	example so that you can quickly get up to speed with no external
 	dependencies.
 
-Generating a Live Report
-------------------------
+``iotile-analytics`` works on Python 2.7+ and Python 3.5+.  There is no usage difference
+on Python 2 or Python 3.
+
+
+Generating Live Reports
+-----------------------
 
 Live Reports are a key part of IOTile Analytics.  They are static HTML documents
 that either embed data and plots or reference local data files that can be
 accessed without needing a web server.
 
+.. important::
+
+	Download :download:`freezer.hdf5 <data/freezer.hdf5>` to use as source data and
+	place it in your current working directory.
+
+The quickest way to generate a live report is to use the `analytics-host` command
+line program that comes with `iotile-analytics-interactive`.  You can use the 
+program to generate reports from datasets that are either stored locally on your
+computer or downloaded on the fly from iotile.cloud.
+
+Almost always, the output of `analytics-host` is an interactive html+javascript
+document embedding graphs, data tables and other widgets to let you scroll through
+your data.  Sometimes the output is just plain text or a different binary format.
+
+Python packages you have installed can register LiveReport types with
+`analytics-host` so that you run them from the command line.  Let's see what
+reports you have installed:
+
+.. code:: bash
+
+	$ analytics-host -l
+	Installed Report Count: 3
+
+	Known Reports:
+	 - save_hdf5: Save all data locally as an HDF5 database file.
```

Comment:
Kind of weird that `save_hdf5` is consider a report. I am guessing it was just convenient to reuse same infrastructure, but it seems weird from a user interface point of view

Reply:
I'm open to renaming what this tool does but generally a LiveReport is supposed to take an AnalysisGroup (right now a device, archive or project stored either locally or in iotile.cloud), do something with it and produce one or more output files.  Often those files are html reports so there is a built-in templating features that make that easy but for example `basic_info` is not (it's just plain text) and save_hdf5 is not, which is why the program is called `analytics-host` and not `iotile-report` but I can change this text to say "Installed Analysis Templates" to make clear that it's not necessarily a "report".